src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml

```diff
@@ -53,9 +59,9 @@
 
     </Grid>
 
-    <GridSplitter Grid.Column="1" />
+    <GridSplitter Grid.Column="1"/>
 
-    <Grid Grid.Column="2" RowDefinitions="*,Auto,*" >
+    <Grid Grid.Column="2" RowDefinitions="*,Auto,*">
```

Comment:
I think we still need IsVisible here, otherwise this part of the layout will be rendered and interactable, even if it technically has width of 0.